docs/guide/structure-controllers.md

```diff
@@ -112,32 +112,32 @@ For this reason, controller IDs are often nouns referring to the types of the re
 For example, you may use `article` as the ID of a controller that handles article data.
 
 By default, controller IDs should contain these characters only: English letters in lower case, digits,
-underscores, dashes and forward slashes. For example, `article` and `post-comment` are both valid controller IDs,
+underscores, hyphens, and forward slashes. For example, `article` and `post-comment` are both valid controller IDs,
 while `article?`, `PostComment`, `admin\post` are not.
 
 A controller ID may also contain a subdirectory prefix. For example, `admin/article` stands for an `article` controller
 in the `admin` subdirectory under the [[yii\base\Application::controllerNamespace|controller namespace]].
-Valid characters for subdirectory prefixes include: English letters in lower and upper cases, digits, underscores and
+Valid characters for subdirectory prefixes include: English letters in lower and upper cases, digits, underscores, and
 forward slashes, where forward slashes are used as separators for multi-level subdirectories (e.g. `panels/admin`).
 
 
 ### Controller Class Naming <span id="controller-class-naming"></span>
 
-Controller class names can be derived from controller IDs according to the following rules:
+Controller class names can be derived from controller IDs according to the following procedure:
 
-* Turn the first letter in each word separated by dashes into upper case. Note that if the controller ID
+1. Turn the first letter in each word separated by hyphens into upper case. Note that if the controller ID
   contains slashes, this rule only applies to the part after the last slash in the ID.
-* Remove dashes and replace any forward slashes with backward slashes.
-* Append the suffix `Controller`.
-* And prepend the [[yii\base\Application::controllerNamespace|controller namespace]].
+1. Remove hyphens and replace any forward slashes with backward slashes.
+1. Append the suffix `Controller`.
+1. Prepend the [[yii\base\Application::controllerNamespace|controller namespace]].
```

Comment:
OK, done. But I'm fairly new to git, so let me know if I got anything wrong while adding that change. (I've been following the instructions at (https://github.com/yiisoft/yii2/blob/master/docs/internals/git-workflow.md), which are super useful, but I may've done something wrong anyway.)

(Side note, just to explain why I did the numbering that way before: I tend to figure that since Markdown ignores the numbers specified, using the "1, 2, 3" format can give a misleading sense of security about what the step numbers will be when rendered. For example, if someone adds a new step in the middle but neglects to renumber the later steps, then hardcoded references like "go to step 4" will be wrong, but that won't necessarily be obvious in the Markdown source. That said, I do recognize the advantages of having readable source, and I'm happy to go with your standard, so I made the change--I'm not arguing against it, just explaining why I did it that way before.)